anydex/test/wallet/ethereum/test_eth_wallet.py

```diff
@@ -41,7 +44,7 @@ def test_create_wallet(self):
         Test the creation of the wallet
         """
         self.wallet.create_wallet()
-        addr = self.wallet._session.query(Key.address).first()[0]
+        addr = self.wallet.database.session.query(Key.address).first()[0]
```

Comment:
Quite a minor detail, but the wallet could be refactored to provide direct access to the database session, as to be compliant with Demeter's Law.

Reply:
I think this is fine, since it's only used like this in tests